name: status check
on: push
jobs:
  fail:
    runs-on: ubuntu-latest
    steps:
      - run: exit 1; #終了ステータスが0以外なのでエラーになる
      - run: echo 'Hello'
        if: ${{ failure() }}
				#エラーが発生しても,failure関数があるので実行される